src/main/java/no/nav/pto/veilarbportefolje/arenaaktiviteter/ArenaAktivitetUtils.java

```diff
@@ -0,0 +1,79 @@
+package no.nav.pto.veilarbportefolje.arenaaktiviteter;
+
+import no.nav.common.types.identer.AktorId;
+import no.nav.common.types.identer.Fnr;
+import no.nav.pto.veilarbportefolje.arenaaktiviteter.arenaDTO.ArenaDato;
+import no.nav.pto.veilarbportefolje.arenaaktiviteter.arenaDTO.ArenaInnholdKafka;
+import no.nav.pto.veilarbportefolje.arenaaktiviteter.arenaDTO.GoldenGateDTO;
+import no.nav.pto.veilarbportefolje.arenaaktiviteter.arenaDTO.GoldenGateOperations;
+import no.nav.pto.veilarbportefolje.domene.AktorClient;
+
+import java.time.ZonedDateTime;
+
+import static no.nav.common.utils.EnvironmentUtils.isDevelopment;
+
+public interface ArenaAktivitetUtils {
+     static <T extends ArenaInnholdKafka> T getInnhold(GoldenGateDTO<T> goldenGateDTO) {
+        switch (goldenGateDTO.getOperationType()) {
+            case GoldenGateOperations.DELETE:
+                return goldenGateDTO.getBefore();
+            case GoldenGateOperations.INSERT:
+            case GoldenGateOperations.UPDATE:
+                return goldenGateDTO.getAfter();
+            default:
+                throw new IllegalArgumentException("Ukjent GoldenGate opperasjon");
+        }
+    }
+
+    static boolean skalSlettesGoldenGate(GoldenGateDTO<?> kafkaMelding) {
+        return GoldenGateOperations.DELETE.equals(kafkaMelding.getOperationType());
+    }
+
+    static boolean erGammelHendelseBasertPaOperasjon(Long hendelseFraDB, Long hendelseFraKafka, String operation) {
+        if(hendelseFraDB == null){
+            return false;
+        }
+        if(hendelseFraKafka == null){
+            return true;
+        }
+        if(GoldenGateOperations.DELETE.equals(operation)){
+            return hendelseFraKafka.compareTo(hendelseFraDB) < 0;
+        }
+        return hendelseFraKafka.compareTo(hendelseFraDB) < 1;
+    }
+
+    static AktorId getAktorId(AktorClient aktorClient, String personident) {
+        try {
+            return aktorClient.hentAktorId(Fnr.ofValidFnr(personident));
+        } catch (RuntimeException exception) {
+            if (isDevelopment().orElse(false)) {
+                return AktorId.of("-1");       // TODO: Diskuter dette.
```

Comment:
I Q1 så skapes det error når vi bruker PDL for å hente fnr -> aktørID.
Dette skaper så mange error at jeg la inn denne linjen for å ignorere feilen